Library/Formula/fish.rb

```diff
@@ -21,6 +21,9 @@ class Fish < Formula
 
   def install
     system "autoconf" if build.head?
+    inreplace "fish.pc.in" do |s|
+      s.gsub! /^completionsdir=.*$/, "completionsdir=#{HOMEBREW_PREFIX}/share/fish/vendor_completions.d"
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!